content/en/blog/2023-01-01-december-update/index.md

```diff
@@ -0,0 +1,330 @@
+
+- <https://github.com/fluxcd/flux2/releases/tag/v0.38.3>
+- <https://github.com/fluxcd/flux2/releases/tag/v0.38.2>
+- <https://github.com/fluxcd/flux2/releases/tag/v0.38.1>
+- <https://github.com/fluxcd/flux2/releases/tag/v0.38.0>
+
+We have released Flux v0.38. Users are encouraged to upgrade for the best
+experience.
+
+Features and improvements
+
+- Graduation of Notification APIs to `v1beta2`, to upgrade please see
+  [the release notes](https://github.com/fluxcd/flux2/releases/tag/v0.38.0).
+- Support for defining Kustomize components with `Kustomization.spec.components`.
+- Support for piping multi-doc YAMLs when publishing OCI artifacts with
+  `kustomize build . | flux push artifact --path=-`.
+- Support for Gitea commit status updates with `Provider.spec.type` set to
+  `gitea`.
+- Improve the memory usage of `helm-controller` by disabling the caching of
+  `Secret` and `ConfigMap` resources.
+- Update the Helm SDK to v3.10.3 (fix for Helm CVEs).
+- All code references to `libgit2` were removed, and the
+  `GitRepository.spec.gitImplementation` field is no longer being honored.
+
+Documentation
+
+The official example repository was refactored. The new version comes with
+the following improvements:
+
+- Make the example compatible with ARM64 Kubernetes clusters.
+- Add Weave GitOps Helm release to showcase the Flux UI.
+- Replace the ingress-nginx Bitnami chart with the official one that contains multi-arch container images.
+- Add cert-manager Helm release to showcase how to install CRDs and custom resources using dependsOn.
+- Add Let’s Encrypt ClusterIssuer to showcase how to patch resources in production with Flux Kustomization.
+- Add the flux-system overlay to showcase how to configure Flux at bootstrap time.
+
+♥ Big thanks to all the Flux contributors that helped us with this release!
+
+### Security news
+
+### Flagger x.y.z
+
+- <https://github.com/fluxcd/flagger/blob/main/CHANGELOG.md#1270>
+
+### Flux Ecosystem
+
+<!--
+
+If you add entries to this subsection, please don't use "we" as it gets
+confusing from which perspective the whole blog post is written and who
+"we" is. The whole post is meant to be from the perspective of the Flux
+community. Better to write:]
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
+#### Terraform-controller
+
+#### Weave GitOps
+
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.15.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.14.1>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.13.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.12.0>
+
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
+#### New additions to the Flux Ecosystem
+
+## Recent & Upcoming Events
+
```

Comment:
@staceypotter @vanessaabankwah Anything we should bring up here for Dec+Jan+Feb?